R/translate-sql-odbc.R

```diff
@@ -35,7 +50,79 @@ base_odbc_agg <- sql_translator(
 base_odbc_win <- sql_translator(
   .parent = base_win,
   n             = function() sql("COUNT(*)"),
-  count         = function() sql("COUNT(*)")
+  count         = function() sql("COUNT(*)"),
+
+  # Window functions in this variant use field that its being
+  # 'windowed' as the secondary ORDER BY instead of the field
+  # from win_current_order().  In the tests, having an ORDER BY
```

Comment:
I'd say the CRAN version is correct because you haven't specified an ordering using `arrange()`. I think the user has to do this themselves, rather than us guessing.